core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -293,42 +307,33 @@ protected String getDefaultUniqueTag() {
             return "aggregator:"+publishing.getName();
         }
         public EnricherSpec<?> build() {
-            Predicate<Object> valueFilter;
```

Comment:
won't removing this break every aggregator which relies on `valueFilter` to deliver `excludingBlank` behaviour?

is it better to keep the if block and fail if both value-filter and excluding blank are set (or construct a new `and` filter) ?